ghdocs/Contributing/Contributing.md

```diff
@@ -0,0 +1,92 @@
+# [office-ui-fabric-react](../../../README.md) › Contributing
```

Comment:
There is already a `CONTRIBUTING.md` in ghdocs. Can you merge these two so we don't have 2 versions?